### DISCUSSION TITLE
Bundle classless version with #root

### DISCUSSION THREAD

**Discussion by dbartholomae** [picocss/pico#315] [Ideas]
Hi there!

So far, every single time I used pico, it was in a React application, and I ended up running into the issue that I would need to recompile it to work with `#root` as root instead of `body`. How about just bundling a `#root` version in the npm package as well? Seems to be a common enough usecase from my perspective :)

**Comment by etdr** [comment]
Bumping this, also to ask if there's a way to recompile without using SCSS..
